README.md

```diff
@@ -102,6 +113,74 @@ Open a python shell in the lms or the cms:
     make lms-shell
     make cms-shell
 
+## For developers
+
+In addition to running Open edX in production, you can use the docker containers for local development. This means you can hack on Open edX without setting up a Virtual Machine. Essentially, this replaces the devstack provided by edX.
+
+First, configure your project such that the LMS and the CMS can be accessed locally:
+
+    make configure
+    ...
+    Your website domain name for students (LMS) (default: "www.myopenedx.com"): localhost:8000
+    Your website domain name for teachers (CMS) (default: "studio.myopenedx.com"): localhost:8001
+    ...
+
+Then, build the images and prepare the database:
+
+    make build
+    make migrate
+
+Point to your local install of [edx-platform](https://github.com/edx/edx-platform/) on your host machine:
+
+    export EDX_PLATFORM_REPOSITORY=/path/to/your/edx-platform
+
+Point to your settings file:
+
+    export EDX_PLATFORM_REPOSITORY=development
```

Comment:
@regisb shouldn't this be `EDX_PLATFORM_SETTINGS` and not `EDX_PLATFORM_REPOSITORY`?

Reply:
Yes, absolutely.